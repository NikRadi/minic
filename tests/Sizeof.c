int main() {
    int x1 = 123;
    printf("%d %d %d\n", sizeof(x1), sizeof x1, sizeof(x1 + 123));
    sizeof(x1 = 3);
    printf("%d\n", x1);

    int *x2;
    printf("%d\n", sizeof(x2));
}
