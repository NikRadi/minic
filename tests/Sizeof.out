4 4 4
123
8